src/Pages/Review/components/TicketCard/TicketCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { addToBasket } from '../../actions';
+import './ticketcard.scss';
+
+export class TicketCardComponent extends Component {
+    constructor (props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleClick = (item) => {
+        this.props.dispatch(addToBasket(item));
+        this.setState({
```

Comment:
Using state for this component seems to be problematic. The component is highlighting correctly when you add it to the basket but the highlighting isn't removed when the item is removed from the basket. The issue here is the basket component wont be able to access the state of this component, therefore you should really shift this out to redux.